src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const itemsList = list.querySelectorAll('li');
+
+itemsList.forEach(li => {
+  if (li) {
+    const span = document.createElement('span');
+
+    li.prepend(span);
+    span.append(span.nextSibling);
+  }
+});
+
+list.addEventListener('click', (e) => {
+  if (!e.target.closest('li')) {
+    return;
+  }
+
+  const item = e.target.closest('li').children[1];
```

Comment:
Instead of using 'e.target.closest('li').children[1]', you can use 'e.target.nextSibling' to improve readability and directly target the element you want to hide or show.